package.json

```diff
@@ -14,11 +14,14 @@
     "pagefind": "^1.1.0",
     "path-to-regexp": "^6.2.2",
     "pdc": "^0.2.3",
-    "remark-gfm": "^4.0.0",
+    "remark-definition-list": "^2.0.0",
     "remark-man": "^9.0.0",
     "remark-parse": "^11.0.0",
     "unified": "^11.0.5",
+    "unist-builder": "^4.0.0",
```

Comment:
package* unrelated changes

Reply:
These are needed for util/generate_man.js.  4 package dependencies were added and 1 was removed.

Reply:
okay, but what about package-lock.json with a billion changes?

Reply:
Those new packages apparently have a billion sub-dependencies.